src/Server.jl

```diff
@@ -74,9 +76,14 @@ struct RequestParams{T<:RequestTrigger}
         target = get(action_kwargs, :target, nothing)
 
         if evt.payload["repository"]["private"] && get(config["registrator"], "disable_private_registrations", true)
-            err = "Private registration request recieved, ignoring"
+            err = "Private registration request received, ignoring"
             @debug(err)
         elseif action_name == "register"
+            # TODO:
+            # - The syntax with which users declare their patch notes is still undecided.
+            # - This assumes that patch notes appear last in the body. Is that fair?
+            patch_match = match(r"Patch notes:(.*)"s, body = get_body(evt.payload))
```

Comment:
You can't use kwarg `body = ` here. I think you meant to set `body` before calling match.

Reply:
Oops, yeah I had initially had `body = ...` in a separate line and  moved it in here. Thanks for catching.